_tools/opengazettes.md

```diff
@@ -1,21 +1,38 @@
 ---
 layout: tool
+name: opengazettes
```

Comment:
The others look like the below, don't they?

-title: Community-Based Monitoring
+name: Community-Based Monitoring